languages/js/test/parity.ts

```diff
@@ -185,5 +185,24 @@ oso.registerClass(E);
   if (((await result.next()).value as Map<string, any>).get('z') !== 3)
     throw new Error();
 
+  // Test lookup in rule head
+
+  // This query should succeed--is that what's happening????
```

Comment:
```suggestion
```

Reply:
So is this actually checking that the query succeeds?

Reply:
yep